core/modules/proto/TaskMsgDigest.cc

```diff
@@ -27,6 +27,10 @@
 // System headers
 #ifdef __SUNPRO_CC
 #include <sys/md5.h>
+#elif __APPLE__
+#define COMMON_DIGEST_FOR_OPENSSL
+#include <CommonCrypto/CommonDigest.h>
+#define MD5 CC_MD5
```

Comment:
Is this #define wanted/needed?  It looks like you define "MD5" as a non-macro below?